sklearn/preprocessing/data.py

```diff
@@ -1325,6 +1325,21 @@ def normalize(X, norm='l2', axis=1, copy=True, return_norm=False):
     return_norm : boolean, default False
         whether to return the computed norms
 
+    Returns
+    -------
+    X : {array-like, sparse matrix}, shape [n_samples, n_features]
+        Normalized input X.
+
+    norms : array
+        When X is sparse, an NotImplementedError will be raised for
+        norm 'l1' or 'l2'; an array with same shape with X.data,
```

Comment:
I meant that it is strange that sparse norms are not just returned per row/column, but for each element.